src/Nordea/Components/FilterLabel.elm

```diff
@@ -0,0 +1,67 @@
+module Nordea.Components.FilterLabel exposing (init, view)
+
+import Css
+    exposing
+        ( alignSelf
+        , border3
+        , borderBox
+        , borderRadius
+        , boxSizing
+        , center
+        , cursor
+        , displayFlex
+        , fitContent
+        , flexDirection
+        , fontFamilies
+        , maxWidth
+        , noWrap
+        , padding2
+        , pointer
+        , rem
+        , solid
+        , whiteSpace
+        , width
+        )
+import Html.Styled as Html exposing (Attribute, Html)
+import Html.Styled.Attributes exposing (css)
+import Nordea.Components.Text as Text
+import Nordea.Resources.Colors as Colors
+import Nordea.Resources.Icons as Icon
+import Nordea.Themes as Themes
+
+
+type FilterLabel
+    = FilterLabel { label : String }
+
+
+init : { label : String } -> FilterLabel
+init { label } =
+    FilterLabel { label = label }
+
+
+view : List (Attribute msg) -> FilterLabel -> Html msg
+view attrs (FilterLabel { label }) =
+    Html.button
+        ([ css
+            [ padding2 (rem 0.25) (rem 0.75)
+            , displayFlex
```

Comment:
Did you test in safari or firefox? I suspect we might need a `appearance: none` to get rid of the default button styles in one of those browsers

Reply:
Tested now. Button is looking fine. The only styling they add is `onClick`. Is that something we want to remove? 🤔